force-app/main/default/chat/lwc/messageHistory/messageHistory.js

```diff
@@ -0,0 +1,28 @@
+import {api, LightningElement} from 'lwc';
+
+export default class MessageHistory extends LightningElement {
+    /**
+     *
+     * @type MessageWrapper[]
+     */
+    @api _messages;
```

Comment:
Why did you expose a private variable here?

Reply:
deleted @api 8d870610e229cc8a420aa5daeddcdb7010ca08a5